MySQL Interface/src/DatabaseInterface/DB_Interface.java

```diff
@@ -0,0 +1,220 @@
+package DatabaseInterface;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * 
+ * @author Sam Travers
+ *
+ */
+public class DB_Interface {
+	
+	//DB CONNECTION VARIABLES
+	static String db_connection = "jdbc:mysql://localhost:3306/languages?verifyServerCertificate=false&useSSL=true";		//languages/
+	static String db_user = "root";
+	static String db_password = "whatpassword?";
+	static Connection myConn;
+	static Statement myStmt;
+	
+	static String[][] subMenu;
+
+	public static void main(String[] args) throws Exception
```

Comment:
I don't think there is any need for a main method here, but it's not a major issue, we can remove that in a later update.